printer_mods/See1807/Voron V0 Exhaust and LED Mod/README.md

```diff
@@ -0,0 +1,38 @@
+# Voron V0 Exhaust and LED Mod
+## _No stink, good light_
+
+Since the lack of suction on the VO bothered me and, in my opinion, printers should always have light, I developed this mod.
+
+- no more stink
+- bright, even light
+- Extraction even without LED lighting
+
+## BOM
+## _Exhaust_
+
```

Comment:
What are the four holes on the back of the exhaust cover?  Is a fan supposed to mount there?  What size?  Please add that information to this README.